rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderFederationTests.java

```diff
@@ -614,7 +614,7 @@ private Federation createFederation(int version) {
                     activations
                 );
             case ERP_FEDERATION_FORMAT_VERSION:
```

Comment:
Should this be renamed to `LEGACY_ERP_FEDERATION_FORMAT_VERSION`?

Reply:
yes, lot of renames must be done. i didnt realize that